docs/integrations/sources/salesforce.md

```diff
@@ -131,6 +138,7 @@ Now that you have set up the Salesforce source connector, check out the followin
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                              |
 |:--------|:-----------|:---------------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------|
+| 2.1.2   | 2023-07-06 | [28781](https://github.com/airbytehq/airbyte/pull/28781) | Fix pagination for BULK API jobs; Addoption to force use BULK API                                                                    |
```

Comment:
nit: only if we're keeping the new option, the changelog is missing a space between `Addoption`